Cargo.toml

```diff
@@ -9,8 +9,9 @@ edition = "2021"
 anyhow = "1.0"
 camino = "1.0"
 clap = { version = "3.0", features = ["derive"] }
-log = "0.4"
-tokio = { version = "1", features = ["full"] }
+tokio = { version = "1", features = ["fs", "io-util", "sync", "macros", "rt-multi-thread"] }
```

Comment:
There is no problem in using `full` here, but specifying the features you want can lead to faster compile times.

Reply:
Good to know! I had the feeling `full` was too much but decided to postpone it. How do you know what `features` to add?

Reply:
I looked into https://docs.rs/crate/tokio/latest/features and added features as needed to make it compile :sweat_smile:

Reply:
> as needed to make it compile 😅

I love that — no kidding :)